frontend/src/__tests__/cypress/cypress/e2e/modelServing/ModelServingGlobal.cy.ts

```diff
@@ -421,6 +515,51 @@ describe('Model Serving Global', () => {
 
     // Submit and check the invalid error message
     inferenceServiceModal.findSubmitButton().click();
+
+    cy.wait('@inferenceServicesError').then((interception) => {
+      expect(interception.request.method).to.eql('POST');
```

Comment:
The interception is for the `POST` method. This check is redundant.

Reply:
Is there any reason why we don't want the check here?

Reply:
```
  cy.intercept(
    {
      method: 'POST',
      pathname: '/api/k8s/apis/serving.kserve.io/v1beta1/namespaces/test-project/inferenceservices',
    },
    { statusCode: 500 },
  );
  ).as('inferenceServicesError');
```

If the `wait` passes, it implies that that we successfully intercepted the aliased request. The aliased request is for a `POST` request. Therefore it is redundant to check that the request if of type `POST` since the interceptor only matches `POST` requests.